aisdc/attacks/likelihood_attack.py

```diff
@@ -32,6 +35,15 @@
 P_THRESH = 0.05  # default significance threshold
 
 
+def parse_boolean_argument(value):
+    """Parses string value to a boolean"""
+    value = value.lower()
+    return_value = False
+    if value in ["true", "True"]:
+        return_value = True
+    return return_value
```

Comment:
There is a good reason that there isn't a bool type specified. The absence of a command line option means that it is not active, i.e., it's false. So passing in `--shadow-models-fail-fast` would mean the user wants it to be true. Adding "true" or "false" after that option is redundant. That's why I suggested the add argument code below, and that means we can drop this function because it's bloat.

Reply:
Thanks for your feedback. 

Yes, I agree; I did not get it at first. 

Now the function is removed.